lib/tenios/api/transfer_call.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Tenios
+  module API
+    class TransferCall
+      EXTERNAL_NUMBER = "EXTERNALNUMBER"
+      SIP_USER = "SIP_USER"
+      SIP_TRUNK = "SIP_TRUNK"
+
+      DESTINATION_TYPES = [
+        EXTERNAL_NUMBER,
+        SIP_USER,
+        SIP_TRUNK
+      ].freeze
+
+      attr_reader :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def transfer(call_uuid:, destination_type: EXTERNAL_NUMBER, destination:)
```

Comment:
Some context on the naming conventions of this API client: All classes methods follow the Tenios API path structure at the moment.

It may be nice to try to do the same on this one, even though this is the first endpoint that doesn't have a "namespace" in the path.

So a couple of suggestions:
1. For consistency with the Tenios API path name, would it be better to name this method the same as the HTTP path, e.g. `transfer_call`?
2. Also, for consistency with the Tenios API path name, would it make sense to add a `transfer_call` method to the Client class that would just do the HTTP request, e.g. in `client.rb`:
```ruby
def transfer_call(...)
  @transfer_call ||= TransferCall.new(self)
  @transfer_call.transfer_call(...)
end
```

Reply:
Updated.

I wasn't really sure what to call the methods for this reason, too - I also considered a more generic name like `call`, but this could get confusing in the context of a library about phones! 😅